src/go/configinfo/service_info.go

```diff
@@ -630,6 +631,39 @@ func (s *ServiceInfo) determineBackendAuthJwtAud(r *confpb.BackendRule, scheme s
 	}
 }
 
+// Apply global setting to all the backends.
+func (s *ServiceInfo) processAllBackends() error {
+	for _, method := range s.Methods {
+		backendInfo := method.BackendInfo
+		if backendInfo == nil {
+			return fmt.Errorf("all the methods should have an un-empty BackendInfo")
+
+		}
+
+		backendInfo.RetryOns = s.Options.BackendRetryOns
```

Comment:
It seems all these "retry" fields are global,  not need to be per "backend" or per "method".   Can we define them in s.Options?

Reply:
I guess the future design is that retry will be configurable in the service config instead of flag, so then it will be per backend. It's good to keep this design.

Reply:
Yes, we would like to be open for the per backend retry policy.